pom.xml

```diff
@@ -45,8 +45,10 @@
 
     <properties>
         <java.version>11</java.version>
-
         <gridsuite-dependencies.version>4</gridsuite-dependencies.version>
+        <postgres.version>13.1.0</postgres.version>
+        <zonky-embedded-database-test.version>2.0.0-beta1</zonky-embedded-database-test.version>
+        <embedded-postgres.version>1.2.10</embedded-postgres.version>
```

Comment:
why use embedded-postgres ? switch to h2 for tests ?

Reply:
When I tested several inheritance strategies with spring data
the generated schema was not compatible with the H2 dialect

Reply:
is it compatible now with JOINs ? which ones were not compatible ? It's good to know probably worth adding in a comment

Reply:
Yes now it is compatible
This is `INHERITS` which was not compatible
```
CREATE TABLE capitals (
    state           char(2)
) INHERITS (cities);
```

Reply:
i 'm going to reuse h2

Reply:
What was the jpa configuration that made it generate inherits statements ?

Reply:
H2 used  now